packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -3225,6 +3228,14 @@ export class ContainerRuntime
 				},
 				error,
 			);
+
+			const testDelayOverrideMs =
+				this.mc.config.getNumber(
+					"Fluid.ContainerRuntime.Test.CloseSummarizerOnSummaryStaleDelayOverrideMs",
+				) ?? defaultRestartDelayMs;
+
+			// Delay 10 seconds before restarting summarizer to prevent the summarizer from restarting too frequently.
```

Comment:
10 seconds seem like a arbitrary and long time. Any reason for picking up 10 seconds? There should be some way of telling from telemetry whether this is high / low and adjust it accordingly. You should consider adding some.

Reply:
Why is 10 seconds problematic?

Reply:
It may not be. But we need data to know that. For example, a busy doc can have thousands of ops in 10 seconds which make the next summary that much more expensive. Or it may turn out that 10 seconds is less and clients keep reloading. It's fine as a starting point but I was interested as to how you came up with 10 seconds? Plus there should be telemetry that tells us whether that number is good enough or not.

Reply:
Do we even need this change? We have the delay from the server as well

Reply:
It's safety against tight loops that could happen if we have some state (that we can't envision) that results in infinite reloads (or possibly - tight reloads for 60 seconds, or something like that). It's arbitrary, and we should reevaluate it once we have more data from PROD. In ideal case we should not hit it :)

Reply:
Not familiar with the summarizer lifecycle; but wondering if waiting 10 seconds here before stopping it VS after, makes a difference.

Reply:
I don't think so since this would be part of the entire submit summary call.